google/resource-snippets/README.md

```diff
@@ -0,0 +1,6 @@
+# Deployment Manager Resource Snippets
+
+This directory contains Google created and managed template for every service. Deployment Manager team will frequently test all snippets/samples under "resource-snippets" folder.
```

Comment:
Rephrase to "This directory contains Deployment Manager templates for demonstrating and testing the configuration of all GCP resources. The Deployment Manager team will test and maintain these resource snippet samples."